### DISCUSSION TITLE
3006.102.2_2-gnuton0_alpha1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#712] [Announcements]
Please provide here feedback about 3006.102.2_2-gnuton0_alpha1.

**Comment by coolsasuke** [comment]
Hi Gnuton,

I have tested this on my GT-BE98, I was using 3006.102.1_1-gnuton0_beta2_rog and updated to 3006.102.2_2-gnuton0_alpha1. Entware seemed to be broken and NTP doesn't sync as well. I had to revert back to previous firmware. Hope you could fix it.

Cheers!

**Comment by poffycat** [comment]
Hello, I have an Asus ROG GTBE98 but I could not install your software in any way. It gives an error while installing. It is not compatible with the device version. What is the reason for this? My device is listed as 2024 production version 2.0. Can you help me?

**Comment by coolsasuke** [comment]
Hi Poffycat,What is your current router firmware version? It is more complicated to install gnuton if you are using latest stock version.Sent from my Galaxy
-------- Original message --------From: poffycat ***@***.***> Date: 30/11/2024  19:51  (GMT+08:00) To: "gnuton/asuswrt-merlin.ng" ***@***.***> Cc: coolsasuke ***@***.***>, Comment ***@***.***> Subject: Re: [gnuton/asuswrt-merlin.ng] 3006.102.2_2-gnuton0_alpha1 (Discussion #712) 
Hello, I have an Asus ROG GTBE98 but I could not install your software in any way. It gives an error while installing. It is not compatible with the device version. What is the reason for this? My device is listed as 2024 production version 2.0. Can you help me?

**Reply by poffycat** [reply]
> Hi Poffycat,What is your current router firmware version? It is more complicated to install gnuton if you are using latest stock version.Sent from my Galaxy -------- Original message --------From: poffycat ***@***.***> Date: 30/11/2024 19:51 (GMT+08:00) To: "gnuton/asuswrt-merlin.ng" ***@***.***> Cc: coolsasuke ***@***.***>, Comment ***@***.***> Subject: Re: [gnuton/asuswrt-merlin.ng] 3006.102.2_2-gnuton0_alpha1 (Discussion #712) Hello, I have an Asus ROG GTBE98 but I could not install your software in any way. It gives an error while installing. It is not compatible with the device version. What is the reason for this? My device is listed as 2024 production version 2.0. Can you help me? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

Yes, I tried downloading the latest and previous software versions from asus website and changing the software version. But nothing changed. It still gives the same error.

**Reply by unisoftenterprises** [reply]
If it's a v2 BE98, then GNUTON is waiting for Asus GPL so likely won't be supported till some time late December at earliest. You can't downgrade stock firmware either and be supported by Asus. Once a v2 is updated to latest stock, the current builds from GNUTON won't apply.

**Reply by poffycat** [reply]
> If it's a v2 BE98, then GNUTON is waiting for Asus GPL so likely won't be supported till some time late December at earliest. You can't downgrade stock firmware either and be supported by Asus. Once a v2 is updated to latest stock, the current builds from GNUTON won't apply.

When can I install the Guniton software? I don't fully understand. Will I be able to install it when GPL arrives? Will this be possible at the end of December or early 2025? I would also like to ask. Is there a hardware difference between version 2 and 1? Can I get information?

**Reply by coolsasuke** [reply]
I did not check if mine was a v2 BE98, but i just got it like 1 month ago. I downgraded stock firmware via rescue mode and upgraded using Gnuton's. Works perfectly. Except that wifi 6ghz channel will be missing after sometime, had to restart the router and it came back again.

**Reply by poffycat** [reply]
> I did not check if mine was a v2 BE98, but i just got it like 1 month ago. I downgraded stock firmware via rescue mode and upgraded using Gnuton's. Works perfectly. Except that wifi 6ghz channel will be missing after sometime, had to restart the router and it came back again.

Can you share which version of the software you reverted to? I want to try it too. I'd appreciate it if you could tell me?****

**Reply by liamstears** [reply]
> > I did not check if mine was a v2 BE98, but i just got it like 1 month ago. I downgraded stock firmware via rescue mode and upgraded using Gnuton's. Works perfectly. Except that wifi 6ghz channel will be missing after sometime, had to restart the router and it came back again.
> 
> Can you share which version of the software you reverted to? I want to try it too. I'd appreciate it if you could tell me?****

Check my post here, it may help:

https://github.com/gnuton/asuswrt-merlin.ng/issues/697#issuecomment-2508587927

**Comment by lastyc** [comment]
Updated from 3006.102.1_1-gnuton0_beta2 to 3006.102.2_2-gnuton0_alpha, Client List (Network Map/View List) stopped displaying once, rebooting solved the problem and it didn't happen again.
I couldn't find any information on what version 3006.102.2_2-gnuton0_alpha is based, on the official 3.0.0.6.102_36855? Thank you for your work!

**Reply by gnuton** [reply]
@lastyc it's based on an old GPL but with security patches.

**Reply by liamstears** [reply]
> @lastyc it's based on an old GPL but with security patches.

Asus has just released a new version yesterday:

ASUS GT-BE98 Firmware version 3.0.0.6.102_36878
1. Strengthened input validation and data processing workflows to further protect information security.
2. Enhanced AiCloud password protection mechanisms, safeguarding against unauthorized access attempts.
3. Enhanced device security through improved buffer handling in connection features.
4. Refined data handling processes, ensuring secure and accurate information management.
5. Enhanced file access control mechanisms, promoting a more secure operating environment.
6. Strengthened certificate protection, providing enhanced data security.

Maybe this is the reason for GPL delay and maybe we will get something soon now

**Comment by EC225** [comment]
Just uploaded the Alpha 1 ROG firmware on my GT BE98 and all went well. I am using MLO for my wifi 7 devices when santa brings them so running to make sure i dont have any problems with legacy devices,

**Comment by EC225** [comment]
I also noticed in the Beta and also the Alpha that the traffic analyzer does not work.

**Comment by EC225** [comment]
Hi everyone I have been running the latest GT BE98 firmware but had a problem and had to restore to original Asus but now, no matter what I do the gnuton firmware will not install. Help please as the stock firmware is not as good in many ways.

**Reply by liamstears** [reply]
If you read above you'll see I already linked to a solution to this problem, you have to use Asus tool to recover the router to an earlier firmware then you can install Gnuton again

Here:
https://github.com/gnuton/asuswrt-merlin.ng/issues/697#issuecomment-2508587927

**Comment by EC225** [comment]
Thank you. Much appreciated

**Comment by EC225** [comment]
I am having a problem at the moment with Wireguard which is running on my GT BE98 (non pro) the wireguard is not following the Director rules and is applying the VPN to all devices. Is this normal or glitch.

I am running Gnuton Alpha

**Reply by EC225** [reply]
I should also add that it works perfectly with open vpn settings

**Comment by Marchcat2008** [comment]
RMerlin wrote how to easily disable the firmware downgrade ban.
This method worked for me without any manipulation using a firmware recovery program.
--------------------------------------------------------------
If flashing on top of Asus stock firmware 3006.102_37000
or greater, then you first need to enable downgrade
capabilities before flashing Asuswrt-Merlin on top
of it.  Over SSH, run the following command:

      nvram set DOWNGRADE_CHECK_PASS=1

After that, you can upload Asuswrt-Merlin through
the webui like any regular firmware upgrade.

This is only required when flashing Asuswrt-Merlin
 for the first time.

**Reply by sayjay09** [reply]
I followed this method and was able to update my BE98 (don't think it's the pro) - thanks! however, as general feedback on the Alpha to you @gnuton there seems to be alot of pages with broken UI's. I had tried both the stock UI and ROG versions and they had the same breaks. 

I've attached an example of the "administration" page. It looks the same on mobile and desktop vivaldi browsers. I had deactivated any tracking and adblocking incase they interfered and nothing had changed. 
![Screenshot_20250104_165353_Vivaldi](https://github.com/user-attachments/assets/b1f75015-9281-49cd-80ea-e8ae60df29e9)

**Reply by Marchcat2008** [reply]
I had the same thing until I cleared my browser cache and everything started displaying correctly.

**Reply by sayjay09** [reply]
Your suggestion worked, thanks 👍 I do notice that some text is missing from the Guest Network Pro options, such as name and WiFi bands (2.4, 5-1, 5-2 or 6GHz).

![Screenshot_20250105_051115_Vivaldi](https://github.com/user-attachments/assets/7560d35d-9ac4-4af6-a603-c50279f69a21)

**Comment by sayjay09** [comment]
@gnuton using DNS over TLS (DoT) with this firmware isn't working but it did with the Asus official build. I've tried disabling and enabling the option but it hasn't made a difference:

![Screenshot_20250105_051723_Vivaldi](https://github.com/user-attachments/assets/d7c1d41f-b9c5-40cf-9551-d78d6b564731)
![Screenshot_20250105_051732_Vivaldi](https://github.com/user-attachments/assets/114f2bf5-f52f-42a9-abf9-0127ff1af9da)

**Reply by Marchcat2008** [reply]
After change firmware restore your router to its factory settings and Initialize all the settings, and clear all the data log for AiProtection, Traffic Analyzer, and Web History.

**Reply by sayjay09** [reply]
I did this and it sorted the missing text, thanks! The DNS Over TLS is not working.
